src/main/java/Eddie/Task.java

```diff
@@ -0,0 +1,46 @@
+package Eddie;
+
+/**
+ * Represents a task which can be created and listed.
+ */
+public class Task {
+    private boolean done;
+    private String name;
+
+    public Task(){};
+    public Task(String name) {
+        this.done = false;
```

Comment:
iirc "done"(booleans) is automatically initialised to false by java compiler so idt you need this statement